ndcube/ndcube.py

```diff
@@ -6,14 +6,18 @@
 import astropy.units as u
 import gwcs
 import numpy as np
-import sunpy.coordinates  # pylint: disable=unused-import  # NOQA
+try:
+    # Import sunpy coordinates if available to register the frames and WCS functions with astropy
+    import sunpy.coordinates  # pylint: disable=unused-import  # NOQA
+except ImportError:
+    pass
 from astropy.wcs.wcsapi import HighLevelWCSWrapper
 from astropy.wcs.wcsapi.wrappers import SlicedLowLevelWCS
 
 from ndcube import utils
 from ndcube.extra_coords import ExtraCoords
 from ndcube.global_coords import GlobalCoords
-from ndcube.mixins import NDCubePlotMixin, NDCubeSlicingMixin
+from ndcube.mixins import NDCubeSlicingMixin, NDCubePlotMixin
```

Comment:
Actually, is there a reason these imports have been reversed?  It's at least a reason the codestyle is complaining, because they are out of alphabetical order.  Although in the case of this PR, the import within the code might also be a cause it's complaining.

Reply:
```suggestion
from ndcube.mixins import NDCubePlotMixin, NDCubeSlicingMixin
```

Reply:
Hm, apparently they do - the tox codestyle is passing on my local branch where I have them reversed, I'll commit that and push it. I didn't think that difference was important, silly me.